Source/JavaScriptCore/runtime/ArrayBufferView.h

```diff
@@ -109,11 +109,16 @@ class ArrayBufferView : public RefCounted<ArrayBufferView> {
             return false;
         return true;
     }
-    
-    virtual JSArrayBufferView* wrap(JSGlobalObject*, JSGlobalObject*) = 0;
-    
+
+    JS_EXPORT_PRIVATE JSArrayBufferView* wrap(JSGlobalObject*, JSGlobalObject*);
+
+    JS_EXPORT_PRIVATE void operator delete(ArrayBufferView*, std::destroying_delete_t);
+
 protected:
-    JS_EXPORT_PRIVATE ArrayBufferView(RefPtr<ArrayBuffer>&&, size_t byteOffset, size_t byteLength);
+    template<typename Visitor> constexpr decltype(auto) visitDerived(Visitor&&);
+    template<typename Visitor> constexpr decltype(auto) visitDerived(Visitor&&) const;
```

Comment:
Seems like these can be private, not just protected.

Reply:
Changed.